database/migration/aruha-1954-subscription-add-field.sql

```diff
@@ -0,0 +1,3 @@
+SET ROLE zalando_nakadi_data_owner;
+
+UPDATE zn_data.subscription SET s_subscription_object = jsonb_set(s_subscription_object,'{updated_at}',concat('"',s_subscription_object->>'created_at','"' )::jsonb, true);
```

Comment:
please don't forget about code formatting in this query.
and also endline at the end of the file.